FlameBot.js

```diff
@@ -38,20 +38,22 @@ class FlameBot {
      * @type {Object}
      */
     this.telegram = telegram;
+
+    /**
+     * The bots username as a Promise
+     * @type {Promise.<string>}
+     */
+    this.usernamePromise = new Promise((resolve, reject) => {
+      telegram.getMe().then((me) => {
+        resolve(me.username);
+      }, reject);
```

Comment:
If I'm not mistaken (and if `getMe()` returns a standard promise) you can leave away the reject part and your promise will automatically be rejected if `getMe` is rejected.